pages/linux/logsave.md

```diff
@@ -0,0 +1,20 @@
+# logsave
+
+> Save the output of a command in a logfile.
+> More information: <https://linux.die.net/man/8/logsave>.
+
+- Execute command with specified argument(s) and save its output to logfile:
+
+`logsave {{logfile}} {{command}}`
```

Comment:
Perhaps it would be good to include some example arguments here. 
I understand that the {{command}} tag would include the command plus the arguments but maybe it would be good to make that explicit.

Reply:
well....maybe that would just make it more confusing. I'm going to approve the PR as I'm not completely sure if we should or shouldn't add these explicit arguments. I'll let you and the other reviewers decide on adding this or not.

Reply:
Thank you @schneiderl for reviewing it. I was debating the same, but as the description clearly says "command with specified argument(s)", I hope this is implied? Just thinking if it would confuse further if we include the arguments for the command? Or should I change `{{command}}` to `{{command_with_arguments}}` or something? Thanks

Reply:
I was thinking about something like `{{command argument}}`. But I think you are right. Adding this would only make it more confusing as it seems like you would be just passing the arguments (at least on the way I've put it). 
I believe we can leave this as is. 
Once again thank you for your contribution.

Reply:
Thanks a lot again.